test/LibraryManager.IntegrationTest/Services/InstallDialogTestService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.Services;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Shell;
+using Microsoft.Web.LibraryManager.Vsix.UI;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest.Services
+{
+    /// <summary>
+    /// Test service for add client side libraries dialog.
+    /// </summary>
+    [Export(typeof(InstallDialogTestService))]
+    public class InstallDialogTestService : VisualStudioTestService
+    {
+        public InstallDialogTestExtension OpenDialog()
+        {
+            Guid guid = Guid.Parse("44ee7bda-abda-486e-a5fe-4dd3f4cefac1");
```

Comment:
>44ee7bda-abda-486e-a5fe-4dd3f4cefac1 [](start = 36, length = 36)

can we refer to the original constants instead of duplicating and hard-coding the guid and cmdId here?